plugins/scaffolder-actions/scaffolder-backend-module-utils/src/actions/zip.test.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2021 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getVoidLogger } from '@backstage/backend-common';
+import { createZipAction } from "./zip"
+import { PassThrough } from 'stream';
+import mock from "mock-fs"
+import fs from "fs-extra"
+
+const mockLogger = getVoidLogger()
+mockLogger.error = jest.fn()
+
+describe("roadiehq:utils:zip", () => {
```

Comment:
Im still not sure about the naming here, but if it has already been agreed by the team, I'm happy. :)

Reply:
Nah the team doesn't agree to it, it's only me, I'll raise this on the standup.